zeeguu_api/api/teacher_dashboard.py

```diff
@@ -343,6 +356,28 @@ def add_teacher(id):
         flask.abort(400)
 
 
+@api.route("/cohort_member_reading_sessions/<id>/<time_period>", methods=["GET"])
+@with_session
+def cohort_member_reading_sessions(id, time_period):
+    '''
+        Returns reading sessions from member with input user id.
+    '''
+    try:
+        user = User.query.filter_by(id=id).one()
+    except sqlalchemy.orm.exc.NoResultFound:
+        flask.abort(400)
+        return "NoUserFound"
+
+    if (not has_permission_for_cohort(user.cohort_id)):
+        flask.abort(401)
+
+    now = datetime.today()
+    date = now - timedelta(days=int(time_period));
+
+    # True input causes function to return context too.
```

Comment:
i can't parse this comment. what does it mean?

Reply:
Ah, it was a leftover from the method I borrowed. removed it